src/redis/cache_impl.go

```diff
@@ -58,6 +59,13 @@ func (this *rateLimitCacheImpl) generateCacheKey(
 	return cacheKey
 }
 
+func max(a uint32, b uint32) uint32 {
```

Comment:
There is no other function like that. I could move this to an utility package, is it worth it if it is not used anywhere else?